Inspicio/ClaimsPrincipalExtensions/ProfileNameClaimsPrincipalFactory.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+    // Jack Lloyd [06/07/17]
+/*
+    Overriding the UserClaimsprincipalFactory
+    in order to add a 'ProfileName' claim 
+    into Claims of Authenticated Users
+ */
+
+using Inspicio.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using System.Security.Claims;
+
+using System.Threading.Tasks;
+
+namespace Inspicio.ClaimsPrincipalExtensions
+{
+    public class ProfileNameClaimsPrincipalFactory : UserClaimsPrincipalFactory<ApplicationUser, IdentityRole>
+    {
+        public ProfileNameClaimsPrincipalFactory (
+            UserManager<ApplicationUser> userManager,
+            RoleManager<IdentityRole> roleManager,
+            IOptions<IdentityOptions> optionsAccessor ) : base( userManager, roleManager, optionsAccessor )
+        {
+
+        }
+
+        public async override Task<ClaimsPrincipal> CreateAsync( ApplicationUser user )
+        {
+
+                // Jack Lloyd [06/07/17]
```

Comment:
Yea, `git blame`. Stupid typos. As for the UI, each of the different git tools have their own way of showing it, but the data is always line-at-a-time. I don't think any git tools have clean UI, it's very much function over form.